code/game/machinery/vending.dm

```diff
@@ -885,14 +893,6 @@
 	ads_list = list("Only the finest!","Have some tools.","The most robust equipment.","The finest gear in space!")
 	refill_canister = /obj/item/vending_refill/assist
 
-/obj/machinery/vending/assist/Initialize(mapload)
```

Comment:
Gonna test a lot of these locally. Modifying consturction calls en masse worries me.

Reply:
As far as I can tell all of them were identical except for the circuit board, so hopefully nothing will have changed.